apps/web/languages/en-US/common.json

```
@@ -60,7 +60,8 @@
     "View My Collection": "View My Collection",
     "View On AlgoExplorer": "View on AlgoExplorer",
     "View Release": "View Release",
-    "View Set": "View Set"
+    "View Set": "View Set",
+    "View image X of Y":"View image X of Y"
```

You'll wanna make this a template translation, like so:

```suggestion
    "View image X of Y":"View image {{x}} of {{y}}"
```

---

fixed (force-push to rebase)